pkg/tools/log/logger.go

```diff
@@ -50,6 +50,8 @@ type Logger interface {
 	Object(k, v interface{})
 
 	WithField(key, value interface{}) Logger
+
+	SetLogLevel(level string)
```

Comment:
If we are introducing such method on the `logger` level, we most probably should additionally create `logger.Level` enum and use it.